test/test_nn.py

```diff
@@ -10607,6 +10607,16 @@ def test_kaiming_normal_errors_on_inputs_smaller_than_2d(self):
                 tensor = self._create_random_nd_tensor(dims, size_min=1, size_max=1)
                 init.kaiming_normal_(tensor)
 
+    def test_kaiming_uniform_warning_on_0element_tensor(self):
+        tensor = self.empty(0, 1)
```

Comment:
I think you want `torch.empty` here and below instead of `self.empty`.

Reply:
Sorry for the delay. Fixed in https://github.com/pytorch/pytorch/pull/56505/commits/baf05333ed74e4581157a483d4800c3613e36ce1.